app/scripts/controllers/attachPVC.js

```diff
@@ -185,18 +193,35 @@ angular.module('openshiftConsole')
               podTemplate.spec.volumes = [];
             }
             podTemplate.spec.volumes.push(newVolume);
-            $scope.alerts = {};
 
             DataService.update(resourceGroupVersion, resource.metadata.name, $scope.attach.resource, context).then(
               function() {
-                $window.history.back();
+                var details;
+                if (!mountPath) {
+                  // FIXME: This seems like a bad experience since we don't give users a way to mount it later.
+                  details = "The volume was added, but not mounted, since no mount path given.";
```

Comment:
can we get an issue open for this? or get it added as a gap in the trello story about pvc experience...  that aside, can you update the text.  Maybe "No mount path was provided. The volume reference was added to the configuration but it will not be mounted into running pods."